browser-test/src/admin_program_viewing.test.ts

```diff
@@ -144,4 +144,69 @@ describe('admin program view page', () => {
 
     await validateScreenshot(page, 'view-program-start-editing')
   })
+
+  it('view program, view multiple blocks, then start editing with long screen name and description', async () => {
+    const {page, adminPrograms, adminQuestions} = ctx
+    await loginAsAdmin(page)
+    await enableFeatureFlag(page, 'esri_address_correction_enabled')
+
+    const programName = 'Apc program'
+    await adminQuestions.addAddressQuestion({questionName: 'address-q'})
+    await adminQuestions.addDateQuestion({questionName: 'date-q'})
+    await adminQuestions.addEmailQuestion({questionName: 'email-q'})
+
+    await adminPrograms.addProgram(programName)
+    await adminPrograms.addProgramBlock(
+      programName,
+      'screen 2 extra looooooooooooooooooooooooooooooooooooooo' +
+      'oooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo' +
+      'oooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo' +
+      'oooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo' +
+      'ooooooooooooooooooooooooçooooooooooooooooooooooooooooooooooooooooooooo' +
+      'ooooooooooooooooooooooooooooooooooooooooooooooooooooooooong description',
+      [],
+    )
+    await adminPrograms.editProgramBlock(
+      programName,
+      'dummy loooooooooooooooooooooooooooooooooooooooooooooooo' +
+      'ooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo' +
+      'ooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo' +
+      'ooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo' +
+      'ooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo' +
+      'oooooooooooooooooooooooooooooooooooooooooooooooooooooooong description',
+      ['address-q', 'date-q', 'email-q'],
+    )
+    await adminPrograms.publishAllDrafts()
+
+    await adminPrograms.gotoViewActiveProgramPage(programName)
+
+    await adminPrograms.gotoToBlockInReadOnlyProgram('1')
+    await adminPrograms.expectReadOnlyProgramBlock('1')
+    await adminPrograms.gotoToBlockInReadOnlyProgram('2')
+    await adminPrograms.expectReadOnlyProgramBlock('2')
+
+    await adminPrograms.expectQuestionCardWithLabel(
+      'address-q',
+      'required question',
+    )
+    await adminPrograms.expectQuestionCardWithLabel(
+      'address-q',
+      'address correction: disabled',
+    )
+    await adminPrograms.expectQuestionCardWithLabel(
+      'date-q',
+      'required question',
+    )
+    await adminPrograms.expectQuestionCardWithLabel(
+      'email-q',
+      'required question',
+    )
+
+    await validateScreenshot(page, 'view-program-block-2-with-long-description')
```

Comment:
I'm not seeing this new snapshot pdf in the PR.

Reply:
Hi I saw multiple of your comments at once, reply is below.

Might be good to add a browser test with a snapshot. 
--> Browser test added, along with the snapshot.

Also, did you see the comments on the issue? What would you think of limiting the number of characters on the name and description fields? 
--> For key issue (making sure delete button is within the container) I think my fix is adequate. Yes I did. Open to adding a limit to the number of characters if you have a number in mind. Though it sounds like a product decision.

I think that we need to limit the max widths for both the block list on the left and for the block info display. Then, we also need to make sure the delete button is always within the container. Honestly, this page needs a major style clean-up, so everything is made a little bit more challenging. I think fixed max-widths would do the trick for now.
 --> I limited the max-width for the block list on the left. It doesn't seem like limiting it for the block info display is necessary given the delete button is always within the container now. 

I'm not seeing this new snapshot pdf in the PR.
--> It's there now.

Reply:
Agree that the character limit should be a product decision.  Let's table that for now.